index.html

```diff
@@ -159,71 +159,110 @@ <h2>
     </section>
     <section> <h3>Permissions and user prompts</h3>
       <p>
-        The term <a>obtain permission</a>
-        is used for acquiring trust for a given operation.
+        The [[PERMISSIONS]] API provides a uniform way for websites to request
+        permissions from users and query which permissions they have.
       </p>
       <p>
-        The term <dfn>expressed permission</dfn> refers to an act by the user, e.g.
-        via user interface or setting or host device platform features, through which
-        the user approves the permission of a <a>browsing context</a> to access the
-        given functionality.
+        A UA may decide that an explicit user permission
```

Comment:
As it reads, this seems not great to be honest... us here should decide which locks require permissions or not, not the UA. Otherwise, we are going to have all sorts of weird behavior across UAs.